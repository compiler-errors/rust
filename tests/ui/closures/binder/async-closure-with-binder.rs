// edition:2021
#![feature(closure_lifetime_binder)]
#![feature(async_closure)]
fn main() {
    for<'a> async || -> () {};
}
