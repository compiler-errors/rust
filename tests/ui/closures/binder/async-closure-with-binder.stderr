error: implicit types in closure signatures are forbidden when `for<...>` is present
  --> $DIR/async-closure-with-binder.rs:5:21
   |
LL |     for<'a> async || ();
   |     -------         ^
   |     |
   |     `for<...>` is here

error: aborting due to 1 previous error

